heroList_GaryMcGinnis.rb

```diff
@@ -0,0 +1,94 @@
+class Heroes
+  @@heroes = []
+  @hero_name = ''
+  @hero_identity = ''
+  @hero_weakness = ''
+
+  def initialize
+      menu
+  end
+
+  def menu
+    choice = 1
+    until choice == 4
+        puts 'What do you want to do?
+              1: Add Hero.
+              2: Read Hero names.
+              3: Delete Hero
+              4: Save File and Quit.'
+        choice = gets.chomp.to_i
+
+        if choice == 1
+          get_hero_name
+          get_hero_identity
+          get_hero_weakness
+          save_hero_list
+
+        elsif choice == 2
+          print_hero_List
+        elsif choice == 3
+          print_hero_List
+          puts 'What Super hero do you want to delete?'
+          delete_choice = gets.chomp
+
+          @@heroes.reject!{|hero| hero[:hero_name].downcase == delete_choice.downcase}
```

Comment:
This seems like domain logic of the class, maybe you can move this into a method?